packages/UI/src/components/atoms/Link/Link.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Story, Meta } from '@storybook/react';
```

Comment:
@alston-hsu 
`ComponentStory` and `ComponentMeta` should be used here instead. please check other components to see 
the way we used those types